uint32 command
uint32 SORT=0
uint32 SINGULATE_CLUTTER=1
uint32 SINGULATE_SORTED=2
uint32 CLUTTER=4
uint32 WAIT=5
uint32 MOVE_HOME=6
uint32 TERMINATE=7

---
bool completed	
uint32 error_code
uint32 OK=1
uint32 NO_CLUSTERS_FOUND=2
uint32 PERCEPTION_ERROR=3
uint32 PROCESSING_ERROR=4
uint32 GRASP_PLANNING_ERROR=5
uint32 PICK_ERROR=6
uint32 PLACE_ERROR=7
uint32 FINAL_MOVE_HOME_ERROR=8
uint32 UNKNOWN_COMMAND_REQUEST_ERROR=10
uint32 UNKNOWN_ERROR=11

